duckduckgo/Android
# ADS: Settings Screen migration

Task/Issue URL: https://app.asana.com/0/0/1203411647299661/f

### Description
Migrate Settings screen and small section to new components

- Main Settings menu
- Theme Dialog
- App Icon Screen
- Fire Button Animation Dialog
- Automatically Clear... Dialog
- Clear On... Dialog
- Open Links in Apps Dialog
- DuckSuckGo Desktop App Screen
- About DuckDuckGo Screen

### Steps to test this PR

- Install from this branch
- Go to Settings
- [ ] Check settings menu, dialogs and screens mention in description look as expected

### UI changes
| Before  | After |
| ------ | ----- |
![Screenshot_20221130-181442_DuckDuckGo](https://user-images.githubusercontent.com/20798495/204876902-c02e41a3-6ff4-40a9-9b04-82cf8206b2d6.jpg)|![Screenshot_20221130-181455_DuckDuckGo](https://user-images.githubusercontent.com/20798495/204876842-55c26244-32c1-4c54-adb6-7572b8862d90.jpg)|
